src/libraries/Microsoft.Extensions.Logging/src/LoggerFactoryScopeProvider.cs

```diff
@@ -48,6 +48,16 @@ void Report(Scope current)
                     }
 
                     callback(activityLogScope, state);
+
+                    if ((_activityTrackingOption & ActivityTrackingOptions.Tags) != 0)
+                    {
+                        callback(activity.Tags, state);
```

Comment:
>Tags [](start = 42, length = 4)

This should be TagObjects and not Tags.

Reply:
I will change it but not sure if it matters, maybe you can elaborate.
To log the values, `ToString()` will be called anyway. Calling `Tags` instead of `TagObjects` does this: https://source.dot.net/#System.Diagnostics.DiagnosticSource/System/Diagnostics/Activity.cs,1450

Reply:
`Tags` support only `KeyValuePair<string, string>` which `TagObjects` is `KeyValuePair<string, object>`. Think about it as `TagObjects` is superset of `Tags`. Also, when requesting `TagObjects` we'll not allocate while requesting `Tags` will force the allocation.